tests/EdFi.Ods.Api.IntegrationTests/EducationOrganizationTestDataBuilder.cs

```diff
@@ -151,6 +155,33 @@ public EducationOrganizationTestDataBuilder AddSchool(int schoolId, int? localEd
             return this;
         }
 
+        public EducationOrganizationTestDataBuilder AddStudent(string newGuidId)
+        {          
+            _sql.AppendLine(
+                $@"INSERT INTO edfi.Student (FirstName,LastSurname,BirthDate,StudentUniqueId)
+                VALUES ('"+ newGuidId + "','" + newGuidId + "','" + DateTime.UtcNow.Date + "','"+ newGuidId + "');"
```

Comment:
We should use string interpolation as best practice

Reply:
fixed it